docs/conf.py

```diff
@@ -0,0 +1,167 @@
+# -*- coding: utf-8 -*-
+#
+# Configuration file for the Sphinx documentation builder.
+#
+# This file does only contain a selection of the most common options. For a
+# full list see the documentation:
+# http://www.sphinx-doc.org/en/master/config
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'Pulp Ansible'
+copyright = '2019, Pulp Ansible Team'
+author = 'Pulp Ansible Team'
+
+# The short X.Y version
+version = '0.1'
+# The full version, including alpha/beta/rc tags
+release = '0.1.0rc5'
```

Comment:
I'm not super familiar with how docs work. Is this something we have to remember to bump when we do a release of pulp_ansible? Is there a way to fetch this from setup.py?

Reply:
I can add this into my next push. Good idea.

Reply:
I messed with this for a bit, but I want to leave it alone. I think instead we should adopt https://pypi.org/project/bumpversion/ which I know a lot of successful projects rely on. How about that for now?

Reply:
Sounds good to me. Maybe file an issue if you feel like it.

Reply:
I made one here  https://pulp.plan.io/issues/4864